game-core/src/test/java/games/strategy/triplea/delegate/battle/casualty/AaCasualtySelectorTest.java

```diff
@@ -511,21 +512,60 @@ void oneTypeOfPlaneWithAmountEqualToDiceSides() {
           is(empty()));
     }
 
+    private DiceRoll givenLowLuckDiceRoll(final List<Unit> aaUnits, final List<Unit> planes) {
```

Comment:
IMHO - a comment on _this_ method would go a long way, what is this method doing?